### PR TITLE
Adds datepicker and custom styles

### LINKED ISSUE #87: ScheduledDate in request modal not formatted correctly
**Describe the bug**
Aeon requires scheduled dates to be in mm/dd/yyyy format, but the hardcoded date picker is currently in yyyy-mm-dd.

**Additional context**
Solution is to turn the field into a text field and create a custom date picker.

Part of #68

### PR DESCRIPTION
**Pull request by helrond**
Converts `date` input to a `text` input with pattern validation and adds a datepicker (https://dequeuniversity.com/library/aria/date-picker) selected for good accessibility (support for keyboard navigation, aria labels, etc.). Adds custom styles to datepicker so it fits in with the rest of the site.

Adding @HaSistrunk as a reviewer for a second look at accessibility.

Fixes #87

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <head>
   <!-- Google Tag Manager -->
-  {% if jekyll.environment == "production" %}
+  {%- if jekyll.environment == "production" -%}
```
Comment (p-galligan) [inline _includes/head.html:3]: Thanks for catching this!

**Review by p-galligan — APPROVED** [review]
Put a request through. Modal looks generally the same, which is nice.

Queue got flagged as "Awaiting Request Processing" and I'm seeing the scheduled date in the transaction.

Looks good!

**Comment by HaSistrunk** [comment]
The one thing I see accessibility-wise is that the input needs an id that links it to the label because currently the input field has no description, only the button. The `name` attribute won't link the label. I'm not sure what this looks like, though. Can the label `for` and the input `name` also be "datepicker" to match the input `id`?

**Comment by HaSistrunk** [comment]
Oh, sorry. The aria-describedby attribute in the button also needs to match the label id to link those two together.

**Comment by helrond** [comment]
The input name needs to be ScheduledDate for Aeon purposes. I can try and change the input id but that was causing weird issues before.

**Comment by HaSistrunk** [comment]
Hm. Are you able to add a label id of `datePickerLabel` to match the aria-describedby in the button?

**Comment by helrond** [comment]
Yes, I should be. Let me look at this a bit more.

**Comment by helrond** [comment]
I've added an id of `datepickerLabel` and a `for` attribute which targets `datepicker` to the label. Current HTML looks like the screencap below.

Screen Shot 2020-09-30 at 1 19 04 PM

I think that gets us what we need?

**Comment by HaSistrunk** [comment]
Yes!

**Pull request merged**